testFramework/unittests/testsListingBox/formatterTest.php

```diff
@@ -105,6 +105,8 @@ public function testListStandardFormatterNoItems()
     public function testListStandardFormatterMultiRow()
     {
         define('TEST_BUY_NOW', 1);
+        define('STORE_STATUS', 1);
+        define('ENABLE_SSL', 1);
```

Comment:
Strictly speaking,you are right. However in terms of the tests passing it was more about having the value defined. 

What this does speak to, is that these classes still need more refactoring, if we want to address Single Responsibility and SOLID programming. 
It would have been better if I could have written a test to address the bug, however the code structure, currently would have made that extremely difficult.